README.md

```diff
@@ -5,7 +5,7 @@ Cookiecutter template to manage consistent creation of Terraform modules.
 ## To use this template
 
 ```bash
-$ pip install cookiecutter hub
+$ pip install -r requirements.txt
```

Comment:
This assumes that the person clones the repo first and that isn't necessary with cookiecutter. More so, cookiecutter will do the cloning for you